language-support/java/codegen/src/main/scala/com/digitalasset/daml/lf/codegen/backend/java/inner/ContractIdClass.scala

```diff
@@ -52,6 +53,18 @@ object ContractIdClass {
             .returns(ClassName.bestGuess(s"$name.ContractId"))
             .build()
         )
+        val tplContractIdClassName = templateClassName.nestedClass("ContractId")
+        idClassBuilder.addMethod(
+          MethodSpec
+            .methodBuilder(s"unsafeFrom$simpleName")
```

Comment:
* [ ] update sample codegen output in documentation

Reply:
Addressed by https://github.com/digital-asset/daml/pull/13460/commits/b883df749b6bfe94f9624001b5c52b728ce5b217